### ISSUE TITLE
include "gene_type" from Ensembl

### ISSUE THREAD

**Issue by newgene** [biothings/mygene.info#28]
Ensembl provides "gene_type" annotations for their Ensembl Genes, they can be retrieved via BioMart query. This is an example to get the mapping from Ensembl Gene ID to "gene_type":

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE Query>
<Query  virtualSchemaName = "default" formatter = "TSV" header = "0" uniqueRows = "0" count = "" datasetConfigVersion = "0.6" >
			
	<Dataset name = "hsapiens_gene_ensembl" interface = "default" >
		<Attribute name = "ensembl_gene_id" />
		<Attribute name = "gene_biotype" />
	</Dataset>
</Query>
```

We can integrate "gene_type" values into the current Ensembl data ingestion process.

**Comment by newgene** [comment]
We can have this new field as "ensembl.gene_type"

**Comment by sirloon** [comment]
Fixed as of commit e988bb3ffbab6be51fc72f1b22a6655bec5b8296
Data including type_of_gene should be available next release this WE (included in upcoming Ensembl release 91)

Mapping is the same as type_of_gene for Entrez:

                "type_of_gene" : {
                        "include_in_all" : false,
                        "index" : "not_analyzed",
                        "type" : "string"
                },

**Comment by sirloon** [comment]
fix type_of_gene under "/ensembl" 08ec1e903111e03fb55ae306031b566f3adf4001
